cpp/CMakeLists.txt

```diff
@@ -218,6 +218,7 @@ add_library(cugraph SHARED
     src/structure/cugraph.cu
     src/link_analysis/pagerank.cu
     src/traversal/bfs.cu
+    src/traversal/sssp.cu
```

Comment:
Does this replace the previous src/traversal/nvgraph_sssp.cu? If yes, can we remove the outdated nvgraph_sssp.cu?

Reply:
Yes, it replaces the nvgraph version. I haven't added the python bindings yet. Once the python bindings are in place, they will naturally call the new code.

Reply:
OK, please don't forget to delete the dead/unused/outdated files once you finished the job. I am still reviewing your code, but you did a great job in writing clean code while keeping the code in sync with the (somewhat convoluted) cuGraph code base. Very impressive!!!

Reply:
Great, yes please make sure to delete the dead code once the python bindings are in place.

Reply:
Done in https://github.com/rapidsai/cugraph/pull/341